edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/ModuleManagementHttpClientTest.cs

```diff
@@ -94,7 +94,7 @@ public async Task ProductInfoTest(string serverApiVersion, string clientApiVersi
         {
             ModuleManagementHttpClient mm = new ModuleManagementHttpClient(this.serverUrl, serverApiVersion, clientApiVersion);
 
-            Assert.Equal("EdgeAgent (kernel_name=foo;cpu_architecture=bar;)", await mm.GetProductInfoAsync(CancellationToken.None, "EdgeAgent"));
+            Assert.Equal("EdgeAgent (kernel=foo;architecture=bar;version=baz;server_version=;kernel_version=;operating_system=;cpus=0;virtualized=;)", await mm.GetProductInfoAsync(CancellationToken.None, "EdgeAgent"));
```

Comment:
Same question about naming and casing.
Should this be as defined for the `edgeAgent_metadata` in this link: https://docs.microsoft.com/en-us/azure/iot-edge/how-to-access-built-in-metrics?view=iotedge-2020-11#available-metrics